poetry.lock

```diff
@@ -31,9 +50,20 @@ yarl = ">=1.0,<2.0"
 [package.extras]
 speedups = ["aiodns", "brotlipy", "cchardet"]
 
+[[package]]
+name = "aiosqlite"
+version = "0.17.0"
+description = "asyncio bridge to the standard sqlite3 module"
+category = "main"
+optional = false
+python-versions = ">=3.6"
+
+[package.dependencies]
+typing_extensions = ">=3.7.2"
+
 [[package]]
 name = "arrow"
-version = "1.1.1"
+version = "1.2.1"
```

Comment:
we shouldn't be updating our dependencies here.

To fix, first revert the commit that updated this file.
Then add the new dependencies with poetry add, or by manually editing pyproject.toml to the currently committed state.
Then, run `poetry lock --no-update`

Now the new dependencies should be in this file without updating every dependency.